xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/DXVAHD.h

```diff
@@ -104,6 +105,7 @@ class CProcessorHD : public ID3DResource
   D3D11_VIDEO_PROCESSOR_RATE_CONVERSION_CAPS m_rateCaps;
   D3D11_TEXTURE2D_DESC m_texDesc;
   PROCESSOR_VIEW_TYPE m_eViewType;
+  std::vector<AVPixelFormat> m_formats;
```

Comment:
how I can decide in CProcessInfo::GetRenderFormats wich format is supported by dxva renderer if I don't know will it be used or not?